src/core_plugins/kibana/server/routes/api/scripts/register_languages.js

```diff
@@ -0,0 +1,27 @@
+import _ from 'lodash';
+import handleESError from '../../../lib/handle_es_error';
+
+export function registerLanguages(server) {
```

Comment:
Not really sure, just following the pattern of the other route registration modules in this plugin.